src/Pages/GeneralInfo.test.js

```diff
@@ -0,0 +1,199 @@
+import { render, screen, fireEvent} from "@testing-library/react";
+import GeneralInfo from '../Pages/GeneralInfo';
+
+
+
+//test to see if general information page is rendered
```

Comment:
Comments like this aren't necessary as the test() method already provides a description of the test